### PR TITLE
docs(website): increase 'get started' button contrast

### LINKED ISSUE #5211: Docs: first homepage "Get Started" button background has low color contrast
### Before You File a Documentation Request Please Confirm You Have Done The Following...

- [X] I have looked for existing [open or closed documentation requests](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3Adocumentation) that match my proposal.
- [X] I have [read the FAQ](https://typescript-eslint.io/docs/linting/troubleshooting) and my problem is not listed.

### Suggested Changes

Screenshot of the Get Started button with blue fill on a dark blue background

Although the button's white text has enough color contrast against the blue button fill, the fill itself isn't very contrasted against the medium/dark blue background. That means users might not know what is or isn't safe to click on (interactive button or non-interactive background).

Suggested remediation: add some kind of outline/shadow to the button. Which is a nice design idea in general, to make it stand out more.

Discussed on stream with @BenDMyers on Twitch: https://www.twitch.tv/videos/1498962341. See https://benmyers.dev/blog/fix-low-contrast-text for a description of low contrast issues.

### Affected URL(s)

https://typescript-eslint.io

### PR DESCRIPTION
**Pull request by andrmaz**
## PR Checklist

-   [x] Addresses an existing open issue: fixes #5211
-   [x] That issue was marked as [accepting prs](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
-   [x] Steps in [CONTRIBUTING.md](https://github.com/typescript-eslint/typescript-eslint/blob/main/CONTRIBUTING.md) were taken

## Overview

https://github.com/typescript-eslint/typescript-eslint/issues/5211

### Light mode
ts-eslint_button-primary-light

### Dark mode
ts-eslint_button-primary-dark

### CODE REVIEW

File: packages/website/src/pages/styles.module.css [hunk 1]
```diff
@@ -33,6 +33,20 @@
   margin: 1rem 1rem;
 }
 
+.button_primary {
```
Comment (Josh-Cena) [inline packages/website/src/pages/styles.module.css:36]: CSS modules use camelCase by convention

Comment (andrmaz) [reply packages/website/src/pages/styles.module.css]: Sure! I have just pushed a new commit to fix that. 
Thanks for reviewing it!

File: packages/website/src/pages/styles.module.css [hunk 2]
```diff
@@ -33,6 +33,20 @@
   margin: 1rem 1rem;
 }
 
+.buttonPrimary {
+  background-color: var(--ifm-color-warning-dark);
+  color: var(--ifm-color-secondary-contrast-foreground);
```
Comment (JoshuaKGoldberg) [inline packages/website/src/pages/styles.module.css:38]: 😕 The gold is pretty jarring given that it's the only not-blue/purple on the page. I have to admit, I'm not a fan of the visual. Could you switch this to just having a thick outline/shadow as suggested by the issue?

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
Thanks for starting on this issue @andrmaz! The gold does help with contrast but I have to admit, I'm partial to sticking with shades of blue. 😄

**Comment by Josh-Cena** [comment]
I started off disliking this. But after staring at it a few times, I have to say it looks quite eye-catching ^^

File: packages/website/src/pages/styles.module.css (hunk 2 above)
Comment (andrmaz) [reply packages/website/src/pages/styles.module.css:38]: @JoshuaKGoldberg Thanks for reviewing it! 
Of course, I'll make this change as soon as I can.

**Review by JoshuaKGoldberg — APPROVED** [review]
Super, thanks!

**Pull request merged**